src/GovUk.Education.ExploreEducationStatistics.Admin.Tests/Services/ReleaseApprovalServiceTests.cs

```diff
@@ -1071,7 +1179,8 @@ await using (var context = InMemoryApplicationDbContext(contextId))
                 contentService ?? Mock.Of<IContentService>(),
                 preReleaseUserService ?? Mock.Of<IPreReleaseUserService>(),
                 releaseFileRepository ?? new ReleaseFileRepository(contentDbContext),
-                releaseFileService ?? Mock.Of<IReleaseFileService>());
+                releaseFileService ?? Mock.Of<IReleaseFileService>(),
+                releaseRepository ?? Mock.Of<IReleaseRepository>());
```

Comment:
You could use strict mocks to highlight any missing setup in the tests or anything being called unexpectedly.